Common/src/main/java/at/petrak/hexcasting/common/loot/AddHexToAncientCypherFunc.java

```diff
@@ -39,17 +40,17 @@ public AddHexToAncientCypherFunc(LootItemCondition[] lootItemConditions) {
      * This doesn't actually have any params so extract behaviour out for the benefit of forge
      */
     public static ItemStack doStatic(ItemStack stack, RandomSource rand) {
-        var fullHex = HexConfig.server().getRandomLootHex(rand);
+        var hex = LOOT_HEXES.get(rand.nextInt(16));
```

Comment:
i know i said hardcode but this should probably be LOOT_HEXES.size() rather than 16. incase we decide to add any more hexes between now and datapacking.